openshift-templates/jenkins-deploy/template.json

```diff
@@ -0,0 +1,353 @@
+{
+  "kind": "Template",
```

Comment:
I need to noodle on whether we want a Jenkins template or to handle the env vars a different way, especially since some of them are secrets anyhow. Right now, this template isn't working correctly as it is colliding with the existing Jenkins template.